### PR TITLE
DM-29983: Add rubintv deployment

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

File: deployments/rubintv/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: events
+
+resources:
+  - github.com/lsst-sqre/rubintv.git//manifests/base?ref=0.0.1
```
Comment (jonathansick) [inline deployments/rubintv/kustomization.yaml:7]: Just make sure to create this `0.0.1` tag before deploying.

**Review by jonathansick — COMMENTED** [review]
This part looks good. Again, be sure to create the tag in rubintv before merging.

File: deployments/rubintv/kustomization.yaml (hunk 1 above)
Comment (jonathansick) [reply deployments/rubintv/kustomization.yaml:7]: Also, is the repo name `rubin-tv`?

Comment (SimonKrughoff) [reply deployments/rubintv/kustomization.yaml:7]: Oh, you're right.  The repo name has the hyphen.

**Comment by jonathansick** [comment]
@SimonKrughoff In addition to the above, you'll also need to make the ArgoCD application resource.

- In `roundtable/deployments/app-land/kustomization.yaml` add `resources/rubintv.yaml`.
- Create the `rubintv.yaml` resource referenced above; it'll be like the one you made for `segwarides`

**Comment by SimonKrughoff** [comment]
OK.  ArgoCD app added.

**Pull request merged**